tests/helpers/boring_model.py

```diff
@@ -197,3 +198,18 @@ def training_step(self, batch, batch_idx):
         self.manual_backward(loss)
         opt.step()
         return loss
+
+
+class ParameterSharingModule(BoringModel):
```

Comment:
can we keep this in the utility tests? I don't think we want other tests to use this because it's not a boring model